tfhub_dev/assets/sayakpaul/arbitrary-image-stylization-inceptionv3-dynamic-shapes/1.md

```diff
@@ -0,0 +1,9 @@
+# Placeholder sayakpaul/arbitrary-image-stylization-inceptionv3-dynamic-shapes/1
+Fast arbitrary image style transfer based on an InceptionV3 backbone.
+
+<!-- dataset: Multiple -->
+<!-- module-type: image-style-transfer -->
+<!-- network-architecture: Other -->
+<!-- fine-tunable: false -->
+<!-- language: en -->
```

Comment:
I think the language model property is not applicable for these models, but let me know if that's not the case.

Reply:
Just fixed it up.